### PR TITLE
Netsurf-fb: new package at 3.10

### PR DESCRIPTION
**Pull request by sdsddsd1**
Please fill out the template.

Read: https://k1ss.org/wiki/kiss/style-guide

---

## Description of package
A lightweight web browser.

## Installed manifest of package

```
/var/db/kiss/installed/netsurf-fb/version
/var/db/kiss/installed/netsurf-fb/sources
/var/db/kiss/installed/netsurf-fb/manifest
/var/db/kiss/installed/netsurf-fb/depends
/var/db/kiss/installed/netsurf-fb/checksums
/var/db/kiss/installed/netsurf-fb/build
/var/db/kiss/installed/netsurf-fb/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/netsurf-fb/quirks.css
/usr/share/netsurf-fb/nl/welcome.html
/usr/share/netsurf-fb/nl/licence.html
/usr/share/netsurf-fb/nl/credits.html
/usr/share/netsurf-fb/nl/
/usr/share/netsurf-fb/netsurf.png
/usr/share/netsurf-fb/ja/welcome.html
/usr/share/netsurf-fb/ja/
/usr/share/netsurf-fb/it/welcome.html
/usr/share/netsurf-fb/it/licence.html
/usr/share/netsurf-fb/it/credits.html
/usr/share/netsurf-fb/it/
/usr/share/netsurf-fb/internal.css
/usr/share/netsurf-fb/icons/show-cookie.png
/usr/share/netsurf-fb/icons/search.png
/usr/share/netsurf-fb/icons/page-info-warning.svg
/usr/share/netsurf-fb/icons/page-info-secure.svg
/usr/share/netsurf-fb/icons/page-info-local.svg
/usr/share/netsurf-fb/icons/page-info-internal.svg
/usr/share/netsurf-fb/icons/page-info-insecure.svg
/usr/share/netsurf-fb/icons/local-history.png
/usr/share/netsurf-fb/icons/hotlist-rmv.png
/usr/share/netsurf-fb/icons/hotlist-add.png
/usr/share/netsurf-fb/icons/directory2.png
/usr/share/netsurf-fb/icons/directory.png
/usr/share/netsurf-fb/icons/content.png
/usr/share/netsurf-fb/icons/arrow-l.png
/usr/share/netsurf-fb/icons/48x48/actions/page-info-warning.png
/usr/share/netsurf-fb/icons/48x48/actions/page-info-secure.png
/usr/share/netsurf-fb/icons/48x48/actions/page-info-local.png
/usr/share/netsurf-fb/icons/48x48/actions/page-info-internal.png
/usr/share/netsurf-fb/icons/48x48/actions/page-info-insecure.png
/usr/share/netsurf-fb/icons/48x48/actions/
/usr/share/netsurf-fb/icons/48x48/
/usr/share/netsurf-fb/icons/24x24/actions/page-info-warning.png
/usr/share/netsurf-fb/icons/24x24/actions/page-info-secure.png
/usr/share/netsurf-fb/icons/24x24/actions/page-info-local.png
/usr/share/netsurf-fb/icons/24x24/actions/page-info-internal.png
/usr/share/netsurf-fb/icons/24x24/actions/page-info-insecure.png
/usr/share/netsurf-fb/icons/24x24/actions/
/usr/share/netsurf-fb/icons/24x24/
/usr/share/netsurf-fb/icons/16x16/actions/page-info-warning.png
/usr/share/netsurf-fb/icons/16x16/actions/page-info-secure.png
/usr/share/netsurf-fb/icons/16x16/actions/page-info-local.png
/usr/share/netsurf-fb/icons/16x16/actions/page-info-internal.png
/usr/share/netsurf-fb/icons/16x16/actions/page-info-insecure.png
/usr/share/netsurf-fb/icons/16x16/actions/
/usr/share/netsurf-fb/icons/16x16/
/usr/share/netsurf-fb/icons/
/usr/share/netsurf-fb/favicon.png
/usr/share/netsurf-fb/en/welcome.html
/usr/share/netsurf-fb/en/licence.html
/usr/share/netsurf-fb/en/credits.html
/usr/share/netsurf-fb/en/
/usr/share/netsurf-fb/default.css
/usr/share/netsurf-fb/de/welcome.html
/usr/share/netsurf-fb/de/
/usr/share/netsurf-fb/ca-bundle
/usr/share/netsurf-fb/adblock.css
/usr/share/netsurf-fb/SearchEngines
/usr/share/netsurf-fb/FatMessages
/usr/share/netsurf-fb/
/usr/share/
/usr/bin/netsurf-fb
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/netsurf-fb/depends [hunk 1]
```diff
@@ -0,0 +1,12 @@
+bison make
+gperf make
+libpng
+libressl
+libwebp
+linux-headers make
+perl make
+#sdl
```
Comment (dylanaraps) [inline community/netsurf-fb/depends:8]: Commented dependency?

**Comment by dylanaraps** [comment]
Very nice. :)

File: community/netsurf-fb/depends (hunk 1 above)
Comment (sdsddsd1) [reply community/netsurf-fb/depends:8]: It is possible to build this frontend with xcb-util or sdl interface.  
Maybe `#sdl - uncomment to build  with additional sdl interface` ?

**Comment by sdsddsd1** [comment]
I have also thought that this might be something for the core repo.

File: community/netsurf-fb/depends (hunk 1 above)
Comment (dylanaraps) [reply community/netsurf-fb/depends:8]: Sure. :+1:

**Pull request merged**